code/addons/highlight/preview.js

```diff
@@ -1 +1 @@
-export * from './dist/esm/highlight';
+export * from './dist/highlight';
```

Comment:
This file won't be created unless it's added to the `bundler.entries` above.

Reply:
I added the file to bundler.entries